Vagrantfile

```diff
@@ -124,6 +124,7 @@ Vagrant.configure(2) do |config|
     config.vm.provision "bootstrap", type: "shell", inline: $bootstrap
     config.vm.provision "build", type: "shell", run: "always", privileged: false, inline: $build
     config.vm.provision "install", type: "shell", run: "always", privileged: false, inline: $install
+    config.vm.box_check_update = false
```

Comment:
I disagree with this being the default behavior - by getting the latest and greatest by default, we get improvements / newer VM versions, etc. I'd prefer if you wrap this with an environment variable, `CHECK_UPDATES=false` where by default, it is `true`.